README.md

```diff
@@ -321,6 +321,36 @@ and add them to the media library if their MIME types are supported. If the
 file/folder in _path_ does not exist/is not readable or is not provided then an
 error will be returned and the corresponding log message would be written into logs.
 
+## Internal Audio & Video Recording
+
+REQUIRED STEPS to activate recording:
+
+```bash
+adb shell pm grant io.appium.settings android.permission.RECORD_AUDIO
+adb shell pm grant io.appium.settings android.permission.WRITE_EXTERNAL_STORAGE
```

Comment:
is this permission still needed?

Reply:
done, removed it https://github.com/sirmordred/io.appium.settings/commit/f0b2628ec6eed40ddf1e880a730747f987d39331, its not needed anymore 💯 as far as i  tested it on both Huawei Android 10 and Samsung Android 12